src/components/modules/nutrition/add-food/food-list-item-info.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div v-if="foodNutrients">
+        <div class="food-info">
+            <div>
+                <span>Protein: </span>
+                <span>{{ foodNutrients.protein.value * portion | cutDigits}}{{ foodNutrients.protein.unit }}</span>
+            </div>
+            <div>
+                <span>Fat: </span>
+                <span>{{ foodNutrients.fat.value * portion | cutDigits }}{{ foodNutrients.fat.unit }}</span>
+            </div>
+            <div>
+                <span>Carbs: </span>
+                <span>{{ foodNutrients.carbs.value * portion | cutDigits }}{{ foodNutrients.carbs.unit }}</span>
+            </div>
+        </div>
+        <v-form v-if="showMore" class="portion-form">
+            <v-text-field
+                v-model="portion"
+                label="Number of portions"
+                type="number"
+            ></v-text-field>
+            <v-icon color="green" class="add-food-icon" @click="addFoodToCurrentDailyIntake">
+                add
+            </v-icon>
+        </v-form>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Watch } from 'vue-property-decorator';
+import { mapGetters } from 'vuex';
+
+import { cutDigits } from '@/filters/CutDigits.filter';
+
+import { FoodNutrient, Food, FoodInfo } from '@/models/nutrition';
+
+@Component({
+    filters: {
+        cutDigits
+    },
+    computed: {
+        ...mapGetters('nutritionModule', {getFoodNutrients: 'foodNutrients'})
+    }
+})
+export default class FoodListItemInfo extends Vue {
+    @Prop({default: null}) foodInfo!: FoodInfo;
+    @Prop({default: false}) showMore!: boolean;
+    @Prop({default: 1}) quantity!: number;
+
+    getFoodNutrients!: Function;
+    foodNutrients: {[key: string]: {value: number, unit: string} } | null = null;
+    portion = this.quantity;
+
+    @Watch('foodInfo') onFoodInfoChange() {
+        if (this.showMore) {
+            this.foodNutrients = this.getFoodNutrients(this.foodInfo.resource_id);
+        } else {
+            this.foodNutrients = null;
+        }
+    }
+
+    addFoodToCurrentDailyIntake() {
+        if (this.foodNutrients) {
```

Comment:
Sometimes it's more clear to use `inverse if` e.g.:

`if (!this.foodNutrients) {
  return;
}
`

Reply:
Done